### PR TITLE
Extend multi selection

### PR DESCRIPTION
**Pull request by handerpeder**
ref: [slack chat](https://clojurians.slack.com/archives/C0185BFLLSE/p1697799626323009)

I often find myself in situations where I'm doing:

```clojure
;; click first arg
(def arg1 @session)
;; click second arg
(def arg2 @session)
(my-func arg1 arg2)
```

this patch is a sketch that would allow the following instead:

```clojure
;; click first arg
;; alt-click second arg
(apply my-func (p/selected session))
```

I've also extended the commands `copy`  and `copy-json` to support multi-selection. If there is only one value selected the value is not wrapped in a list to remain backwards compatible.

At this stage I'm mostly looking for feedback if this is a good approach or not.

To end I'd just like to say that I've found portal a very useful addition to my workflow. This is my first time diving into the code and I found the code to be very clear and well structured. Kudos!

### CODE REVIEW

**Comment by djblue** [comment]
Hi @handerpeder, thanks for the PR!

I forgot to mention a couple of the other clients that need to be updated:

- [node](https://github.com/djblue/portal/blob/master/src/portal/runtime/node/client.cljs#L64)
- [clr](https://github.com/djblue/portal/blob/master/src/portal/runtime/clr/client.clj#L58)
- [web](https://github.com/djblue/portal/blob/master/src/portal/runtime/web/client.cljs#L25)

Do you mind updating them as well?

File: src/portal/api.cljc [hunk 1]
```diff
@@ -235,3 +235,8 @@
    (open (assoc options :value value))))
 
 (register! #'inspect)
+
+(defn selected
+  "Get a sequence of all currently selected values."
```
Comment (djblue) [inline src/portal/api.cljc:240]: Do you mind adding `{:added "0.49.0"}`?

**Comment by handerpeder** [comment]
Sure, was expecting that. I take it this means you're okay with the changes?

**Comment by djblue** [comment]
@handerpeder I think I have a fix for the tests, but looking into some performance issues.

**Comment by djblue** [comment]
A rebase fixed the perf issue, not sure where the regression was but latest seems to be fine 👌 I manually pushed the commits on master https://github.com/djblue/portal/commit/d617feffb26869ea9347118580cc219e26727b73, thanks again for the PR!

**Pull request closed without merging**